Jenkinsfile.integration

```diff
@@ -37,8 +45,33 @@ pipeline {
                 sh 'printenv'
             }
         }
+        stage('Check for updated files') {
+            when { expression { env.BRANCH_NAME != 'master' } }
+            steps {
+                script {
+                    /* Need to fetch master to check against it for the proper diff */
+                    sh "git config --add remote.origin.fetch +refs/heads/master:refs/remotes/origin/master"
+                    sh "git fetch --no-tags"
+                    List<String> sourceChanged = sh(returnStdout: true, script: "git diff --name-only origin/master..origin/${env.BRANCH_NAME}").split()
+                    echo "Files changed for this PR:\n${sourceChanged.join('\n')}"
+                    for (int i = 0; i < sourceChanged.size(); i++) {
+                        if (sourceChanged[i].contains("playbooks/")) {
+                            isPlaybooksChanged = true
+                        }
+                        if (sourceChanged[i].contains("doc/")) {
+                            isDocsChanged = true
+                        }
+                        if (sourceChanged[i].contains("tox.ini")) {
+                            isToxChanged = true
+                        }
+                    }
+                }
+            }
+        }
 
         stage('Lint Ansible playbooks') {
+            when { expression { isPlaybooksChanged == true } }
```

Comment:
Im not sure but this may be changed into `when { expression { return isPlaybooksChanged } }` and work the same

Reply:
this works so I changed it